.Gruntfile.babel.js

```diff
@@ -133,7 +126,7 @@ module.exports = (grunt) => {
       // https://github.com/standard/standard/issues/750#issuecomment-379294276
       eslint: 'node ./node_modules/eslint/bin/eslint.js "**/*.{js,vue}"',
       eslintgrunt: "./node_modules/.bin/eslint --ignore-pattern '!.*.js' .Gruntfile.babel.js Gruntfile.js",
-      stylelint: 'node ./node_modules/stylelint/bin/stylelint.js "frontend/**/*.{css,scss,vue}"',
+      stylelint: 'node ./node_modules/stylelint/bin/stylelint.js "frontend/style/**/*.{css,scss,vue}"',
```

Comment:
This should be `frontend/assets/style/`

Reply:
*Note: I've fixed this and some other things here: https://gitlab.okturtles.org/okturtles/group-income-simple/merge_requests/11/diffs?commit_id=6eb19009aa2c64622707843c73ec8a19f6e4a12d